external/upstream/gauxc/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ if(${ENABLE_gauxc})
 
     if(${gauxc_FOUND})
         #== check if GPU-enabled GauXC is found, if GPU is specified ==#
-        get_property(_gpu TARGET gauxc::gauxc PROPERTY GAUXC_ENABLE_CUDA)
```

Comment:
Instances of user-space code should use `GAUXC_HAS_DEVICE`, not `GAUXC_HAS_CUDA`. The device APIs are the same for CUDA/HIP (and SYCL in a local dev branch).

Reply:
In fixing this suggestion, I realize that I forgot to add `GAUXC_HAS_DEVICE` to the list of exported GauXC properties. I shall update here once that is done.

Reply:
Done and done!